src/custom/components/Version/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useActiveWeb3React } from 'hooks'
+import { ExternalLink, TYPE } from 'theme'
+import { GP_SETTLEMENT_CONTRACT_ADDRESS } from 'custom/constants'
+import { getEtherscanLink } from 'utils'
+
+import { version as WEB, dependencies } from 'src/../../package.json'
+
+const VERSIONS = {
+  WEB,
+  CONTRACTS: dependencies['@gnosis.pm/gp-v2-contracts']
+}
+
+const versionsMap = new Map(Object.entries(VERSIONS))
+const versionsList = Array.from(versionsMap)
```

Comment:
Why, this is the same
```suggestion
const versionsList = Object.entries(VERSIONS)
```